sites/public/pages/listings.tsx

```diff
@@ -167,6 +167,9 @@ const ListingsPage = () => {
     if (filterState.minRent !== undefined && filterState.maxRent != undefined) {
       numberOfFilters -= 1
     }
+    if (filterState.includeNulls) {
+      numberOfFilters -= 1
+    }
```

Comment:
Instead of having special handling here and in `filters.ts`, in `onSubmit` you could have one check like:
```
if (data?.includeNulls === false) {
  delete data.includeNulls
}
```
That will remove the prop from the object, and you shouldn't need to handle it anywhere else.

Reply:
I'd access it via `data?[FrontendListingFilterStateKeys.includeNulls]` instead, though, so it uses the variable that holds the key name instead of the key name directly.

Reply:
Oh but nothing else in this function is doing that, so up to you.